README.md

```diff
@@ -46,18 +46,18 @@ Developers can include the Java AMQP 0-9-1 Client using the following maven depe
 
 # Running the WebSocket demo
 
-## From the command-line
+## Run From the Command-Line
 0. Change directory: ```cd ws/demo/target```
```

Comment:
https://en.wikipedia.org/wiki/Command-line_interface. Unless you have a Chicago Style Guide link for me? :)